### PR TITLE
[DNM] use cluster name instead of id

### PR DESCRIPTION
**Pull request by tiancaiamao**

### CODE REVIEW

File: cmd/pd-server/main.go [hunk 1]
```diff
@@ -21,15 +21,15 @@ var (
 	leaderLease   = flag.Int64("lease", 3, "leader lease time (second)")
 	logLevel      = flag.String("L", "debug", "log level: info, debug, warn, error, fatal")
 	pprofAddr     = flag.String("pprof", ":6060", "pprof HTTP listening address")
-	clusterID     = flag.Uint64("cluster-id", 0, "cluster ID")
+	clusterName   = flag.String("cluster-name", "test", "cluster Name")
```
Comment (siddontang) [inline cmd/pd-server/main.go]: s/cluster-name/name/

File: cmd/pd-server/main.go [hunk 2]
```diff
@@ -21,15 +21,15 @@ var (
...
 	maxPeerNumber = flag.Uint("max-peer-num", 3, "max peer number for the region")
 )
 
 func main() {
 	flag.Parse()
 
-	if *clusterID == 0 {
-		log.Warn("cluster id is 0, don't use it in production")
+	if *clusterName == "test" {
+		log.Warn("cluster id is test, don't use it in production")
```
Comment (siddontang) [inline cmd/pd-server/main.go]: cluster name

File: server/command.go [hunk 3]
```diff
@@ -98,6 +98,99 @@ func (c *conn) getRaftCluster() (*raftCluster, error) {
...
+	if request == nil {
+		return nil, errors.Errorf("invalid get meta command, but %v", req)
+	}
+
+	cluster, err := c.getRaftCluster()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	meta, err := cluster.GetMeta()
```
Comment (siddontang) [inline server/command.go]: s/GetMeta/GetConfig/

File: server/command.go [hunk 4]
```diff
@@ -98,6 +98,99 @@ func (c *conn) getRaftCluster() (*raftCluster, error) {
...
+		return nil, errors.Errorf("invalid get meta command, but %v", req)
+	}
+
+	cluster, err := c.getRaftCluster()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	meta := request.GetCluster()
+	if err = cluster.PutMeta(meta); err != nil {
```
Comment (siddontang) [inline server/command.go]: PutMeta/PutConfig

**Comment by siddontang** [comment]
I think changing all clusterName -> name is ok, @qiuyesuifeng

**Comment by tiancaiamao** [comment]
In struct Cluster's fields, using Name is ok, otherwise clusterName tend to unambiguous.
@siddontang

**Comment by tiancaiamao** [comment]
PTAL

**Pull request closed without merging**